### ISSUE TITLE
403 forbidden when fetching calendars

### ISSUE THREAD

**Issue by Armael** [robur-coop/caldav#35]
Testing locally with a `git daemon` server serving `/tmp/cal` (`git daemon --verbose --export-all --enable=receive-pack`)

```
$ ./_build/default/caldav --admin-password=coolpwd --host=localhost --http=8080 --remote=git://localhost:9418/tmp/cal --tofu
2024-01-28 20:09:09 +01:00: INF [tcpip-stack-socket] Dual IPv4 and IPv6 socket stack: connect
2024-01-28 20:09:09 +01:00: ERR [webdav.fs] error while getting properties for /principals/root/ Cannot find the key /principals/root/.prop
2024-01-28 20:09:09 +01:00: INF [push] { result= Ok done; commands= [ok refs/heads/main]; }
2024-01-28 20:09:09 +01:00: INF [push] { result= Ok done; commands= [ok refs/heads/main]; }
2024-01-28 20:09:09 +01:00: ERR [webdav.robur.io] make_dir: principals do not exist for aces (ids: )
2024-01-28 20:09:09 +01:00: INF [push] { result= Ok done; commands= [ok refs/heads/main]; }
2024-01-28 20:09:09 +01:00: INF [http.server] listening on 8080/HTTP
```

```
$ curl -v -u root:coolpwd -X PUT "http://localhost:8080/users/armael?password=pwd"
< HTTP/1.1 201 Created
$ curl -v -u armael:pwd http://localhost:8080/calendars/armael
< HTTP/1.1 403 Forbidden
$ curl -v -u root:coolpwd http://localhost:8080/calendars/root
< HTTP/1.1 403 Forbidden
## no logs on the server side following the 403s
```

**Comment by birlorg** [comment]
It seems the calendars are not created for you, you have to create them yourself.

**Comment by reynir** [comment]
This is related to #44 I think

**Comment by hannesm** [comment]
With https://git.robur.coop/robur/git-kv/pulls/2 merged and released as git-kv 0.1.0, this issue should no longer be the case. Our binaries are updated, please help yourself with a fresh one.

In case you end up in the same situation, please reopen this issue.
